src/types/Firestore.ts

```diff
@@ -65,7 +65,7 @@ export interface FirestoreData {
   currentVenueNG?: AnyVenue;
   eventPurchase?: Record<string, Purchase>;
   events?: Record<string, VenueEvent>;
-  experiences: Record<string, Experience>;
+  experiences: Record<string, Record<string, Table>>;
```

Comment:
Can you please explain why this change? If we aren't using `Experience` here, should it be used anywhere anymore? What's the layout of this data structure?